src/commands/init/ai-rules.ts

```diff
@@ -0,0 +1,447 @@
+import { resolve } from 'node:path'
```

Comment:
part of code here is duplication, but we still not setelled the command and want to see how it feels so then we can extract stuff into common utils after we work on making command fully public